.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# **Problem:**
+
+- Describe the specific technical problem that this PR solves.
+- Include  details such as:
+    - How you reproduced the problem
+    - What code is affected
+    - Any relevant background information
+
+# **Solution:**
+
+- Describe the specific technical solution that this PR provides.
+- Include details such as:
+    - How you debugged the problem
+    - Why you chose this solution
+    - Does the solution make any impact beyond the immediate problem
+    - Was there any necessarytechnical debt incurred
+    - Any important design decisions/links todesign documentation
```

Comment:
Wonder if it might be helpful here to add linking out to specific sections of the updated code to relevant changes to review. Sometimes a PR is quite large and not all the changes (syntax, formatting) might be relevant to the problem?

Reply:
My thought is that this can be a high level template - let's leave some extras to the person contributing the PR to add